src/common/libidset/idset.h

```diff
@@ -28,6 +28,8 @@ enum idset_flags {
     IDSET_FLAG_BRACKETS = 2, // encode non-singleton idset with brackets
     IDSET_FLAG_RANGE = 4,    // encode with ranges ("2,3,4,8" -> "2-4,8")
     IDSET_FLAG_INITFULL = 8, // initilize/grow idset with all ids set
+    IDSET_FLAG_COUNT_BY_ITER = 16, // disable running count, which speeds up
+                             //  idset_set/clear, but slows down idset_count()
```

Comment:
Another name for this flag could be `IDSET_FLAG_COUNT_LAZY` (only compute count when required). Meh, it is just slightly shorter.

Reply:
I like it!